clang/lib/StaticAnalyzer/Core/BugReporterVisitors.cpp

```diff
@@ -2565,21 +2565,20 @@ class PRValueHandler final : public ExpressionHandler {
   using ExpressionHandler::ExpressionHandler;
 
   Tracker::Result handle(const Expr *E, const ExplodedNode *InputNode,
-                         const ExplodedNode *ExprNode,
+                         const ExplodedNode *RVNode,
                          TrackingOptions Opts) override {
-    if (!E->isPRValue())
-      return {};
+    assert(RVNode->getStmtForDiagnostics() == E &&
```

Comment:
Actually, this will crash if `RVNode` is a `nullptr`, which is only checked afterwards.

Reply:
Moved nullptr check before call.